src/_P073_7DGT.ino

```diff
@@ -1215,7 +1216,7 @@ bool p073_plugin_write_7dt(struct EventStruct *event, const String& text) {
       }
       else {
         if ((p073_temptemp < 100) && (p073_temptemp > -10)) {
-          p073_temptemp    = int(p073_temptemp * 10);
+          p073_temptemp    = round(p073_temptemp * 10.0);
```

Comment:
Please use `10.0f` or else it is initially a `double`

Reply:
`p073_temptemp` is of type `double`, that's why I didn't add the `f`.

Reply:
Ah that's the problem of not seeing all when reading diffs :)